site/src/sphinx/advanced-structured-logging.rst

```diff
@@ -97,6 +122,34 @@ Note that :api:`RequestLogAvailability` is specified when adding a listener.
 :api:`RequestLogAvailability` is an enum that is used to express which :api:`RequestLog` properties
 you are interested in. ``COMPLETE`` will make your listener invoked when all properties are available.
 
+On the client side, you can get the start time and duration information about a connection. Unlike
+request and response properties, you need a different way to get the properties. You should call the static
+method of :api:`ClientConnectionTimings` as follows:
+
+.. code-block:: java
+
+    import com.linecorp.armeria.client.ClientConnectionTimings;
+    import com.linecorp.armeria.client.HttpClient;
+    import com.linecorp.armeria.client.HttpClientBuilder;
+
+    final HttpClient client = new HttpClientBuilder("http://armeria.com")
+            .decorator((delegate, ctx, req) -> {
+                ctx.log().addListener(
+                        log -> {
+                            final ClientConnectionTimings timings = ClientConnectionTimings.get(log);
+                            if (timings != null) {
+                                System.err.println("Duration: " + timings.connectionAcquisitionDurationNanos());
+                            }
+                        }, RequestLogAvailability.REQUEST_START); // Can get after a request is started.
+                return delegate.execute(ctx, req);
+            })
+            .build();
+
+.. note::
+
+    The reason why we used the static method is that the :api:`ClientConnectionTimings` is stored using
+    the attribute. If you want to know more about it, please refer to :ref:`advanced-custom-attribute`.
```

Comment:
See :ref:... for more information. (Remove 'If you want to know more about it'.)